internal/boxcli/cloud.go

```diff
@@ -53,21 +53,15 @@ func cloudShellCmd() *cobra.Command {
 
 func cloudPortForwardCmd() *cobra.Command {
 	command := &cobra.Command{
-		Use:   "port-forward <local-port>:<remote-port> | <port> | :<remote-port> | terminate",
+		Use:   "forward <local-port>:<remote-port> | :<remote-port> | stop | list",
```

Comment:
`ls` instead of `list`?

Reply:
We accept both but the official command is `list` and `ls` is an alias. Do you want me to flip?